### ISSUE TITLE
google feed not working

### ISSUE THREAD

**Issue by AhmedElhosain** [amirzaidi/Launcher3#567]
google feed nor the weather is working after last update with the bridge installed
nokia 6 android 8.1

**Comment by drhwait** [comment]
I also on Nokia 7 plus but bridge not installed

**Comment by Trak-X** [comment]
Same for me on Pixel 2XL. Pixel bridge not installing.

**Comment by misamestanin** [comment]
Same here on my nokia 6 with bridge. Google feed and weather in at glance not working. All permisions are on,location also but nothing. Maybe system launcher 3 and Amir's launcher are in conflict? Version 3.7 without bridge works great.

**Comment by omerdvd** [comment]
Same on LG G6. Bridge not installed.

**Comment by ghost** [comment]
You have to install Rootless Pixel Bridge : https://goo.gl/867Y25

**Comment by omerdvd** [comment]
I know. But on LG G6, the bridge cannot be installed. I keep getting an error: "App not installed."

**Comment by vanillajonathan** [comment]
When tried to install the bridge from within the app, I kept getting the error "App not installed".
However, downloading manually and installing the bridge from APKMirror worked. Then I could just toggle on the option to "Display Google app" in Home Settings.

**Comment by rajithpera** [comment]
I have a same problem. ( Nokia 6.1, Oreo 8.1, aug security patch) 

Pixel bridge installed ( v.2)

**Comment by misamestanin** [comment]
I belive it's issue is on nokia phones only. Bridge app not working. All versions before 3.8 is work great.

**Comment by sidhiqe0001** [comment]
Same for me on OnePlus 5, Pixel bridge not installing. 'App not Installed'. Anybody got solutions.

**Comment by AhmedElhosain** [comment]
the issue had been solved by itself, google feed worked again suddenly

**Comment by rajithpera** [comment]
Mine still not working

**Comment by misamestanin** [comment]
On telegram Amir is give confirmation to my words. Nokia users can't use his launcher(3.8 with bridge)because system launcher and rootless launcher is a conflict. So,only to root yours devices and deinstaled system launcher. Than it will be work. I don't won't to do this so bye bye Amir. My next choice is Lawnchair.

**Comment by rajithpera** [comment]
Seriously...!!! Ok ... I'm gone use 3.6 ( old version)

**Comment by misamestanin** [comment]
3.7 is work perfectly but no updates in future. This is a problem.

**Comment by amirzaidi** [comment]
Weird that 3.7 is installing but the bridge is not, and that Nokia issue seems to only exist on Pie. Can you guys check if uninstalling 3.7 first works?

**Comment by misamestanin** [comment]
On my nokia 6 (8.1android)version 3.7 works great without bridge. Problem is version 3.8.And yes,I do that. First uninstall 3.7 than try 3.8 with bridge and nothing. Not work. Google feed,weather in at glance not work. Plus,when long press at glance widget(calendar only)not open settings.

**Comment by AhmedElhosain** [comment]
for me nokia 6 2017 android 8.1 ... version 3.8 bridge could be installed with no problems in installation but google feed and weather at glance was not working and after awhile it worked itself

**Comment by misamestanin** [comment]
Work perfectly? You have system launcher 3 on your nokia? No root?

**Comment by AhmedElhosain** [comment]
yes it works perfectly for now, and yes there is no root and I have the system launcher 3, Its pre installed and can't be removed normally I think
